source/transactions/tests/retryable-commit-errorLabels.yml

```diff
@@ -0,0 +1,261 @@
+runOn:
+    -
+        minServerVersion: "4.3.0"
+        topology: ["replicaset", "sharded"]
+
+database_name: &database_name "transaction-tests"
+collection_name: &collection_name "test"
+
+data: []
+
+tests:
+  - description: commitTransaction fails with RetryableWriteError label after two network exceptions
+    clientOptions:
+      retryWrites: false
+    failPoint:
+      configureFailPoint: failCommand
+      mode: { times: 2 }
+      data:
+        failCommands: ["commitTransaction"]
+        closeConnection: true
+    operations:
+      - name: startTransaction
+        object: session0
+      - name: insertOne
+        object: collection
+        arguments:
+        session: session0
+        document:
+            _id: 1
+        result:
+        insertedId: 1
+      - name: commitTransaction
+        object: session0
+        result:
+        errorLabelsContain: ["RetryableWriteError", "UnknownTransactionCommitResult"]
```

Comment:
If the driver adds "RetryableWriteError" for a network error, is there any reason this needs to be in a separate test file requiring a 4.3+ server? Related to my comments on the `insertOne` tests, I think we only need to require that server version when using the `errorLabels` fail point option.

If that's the case, then perhaps we can remove this test in favor of incorporating "RetryableWritesError" label assertions into existing commit tests. "add unknown commit label to connection errors" might be one such test.